lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -352,6 +369,137 @@ describe('ContentSelector', () => {
     });
   });
 
+  describe('OneDrive picker', () => {
+    function clickOneDriveButton(wrapper) {
+      const btn = wrapper.find('LabeledButton[data-testid="onedrive-button"]');
+      interact(wrapper, () => {
+        btn.props().onClick();
+      });
+    }
+
+    beforeEach(() => {
+      const picker = FakeGooglePickerClient();
+      picker.showPicker.resolves({
+        id: 'doc1',
+        url: 'https://files.google.com/doc1',
+      });
+      picker.enablePublicViewing.resolves();
+
+      // Silence errors logged if showing OneDrive Picker fails.
+      sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      console.error.restore();
+    });
+
+    it('skips initialization of OneDrive Picker client if parameters not provided', () => {
+      fakeConfig.filePicker.microsoftOneDrive = {
+        enabled: true,
+      };
```

Comment:
To clarify, this doesn't initialize the picker because the parameters _other than enabled_ are missing?

Reply:
Correct.